task_2/variable_var_let.ts

```diff
@@ -40,6 +40,12 @@ secondVariable = true;
 
 //once you assign a value to a constant variable, you cannot change it later
 
+let email;
```

Comment:
@VitaliiMalatok if a variable is defined, you must add the type

Reply:
fixed

Reply:
@VitaliiMalatok type?

Reply:
fixed